### PR TITLE
Wizard recipe: URCU-v0.13.0

### PR DESCRIPTION
**Pull request by hros**
This pull request contains a new build recipe I built using the BinaryBuilder.jl wizard:

* Package name: URCU
* Version: v0.13.0

@staticfloat please review and merge.

### CODE REVIEW

File: U/URCU/build_tarballs.jl [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    Platform("x86_64", "linux"; libc = "glibc"),
+    Platform("armv7l", "linux"; call_abi = "eabihf", libc = "glibc"),
+    Platform("powerpc64le", "linux"; libc = "glibc"),
+    Platform("i686", "linux"; libc = "musl"),
+    Platform("x86_64", "linux"; libc = "musl"),
+    Platform("armv7l", "linux"; call_abi = "eabihf", libc = "musl"),
+    Platform("x86_64", "macos"; ),
+    Platform("x86_64", "freebsd"; )
+]
+
```
Comment (giordano) [inline U/URCU/build_tarballs.jl:36]: Can we build for all platforms and see what happens?
```suggestion
platforms = supported_platforms(; experimental=true)
```

File: U/URCU/build_tarballs.jl [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://git.lttng.org/userspace-rcu.git", "c19d74445b6ed2e6840038afc2f974e3e098334b")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+cd userspace-rcu/
```
Comment (giordano) [inline U/URCU/build_tarballs.jl:16]:
```suggestion
cd $WORKSPACE/srcdir/userspace-rcu/
```

File: U/URCU/build_tarballs.jl [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+./bootstrap 
+./configure --prefix=${prefix} --build=${MACHTYPE} --host=${target}
+make
```
Comment (giordano) [inline U/URCU/build_tarballs.jl:19]:
```suggestion
make -j${nproc}
```

File: U/URCU/build_tarballs.jl [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+# The products that we will ensure are always built
+products = [
+    LibraryProduct("liburcu-memb", :urcu_memb),
+    LibraryProduct("liburcu-mb", :urcu_mb),
+    LibraryProduct("liburcu-qsbr", :urcu_qsbr),
+    LibraryProduct("liburcu-signal", :urcu_signal),
+    LibraryProduct("liburcu-cds", :urcu_cds),
+    LibraryProduct("liburcu", :urcu),
+    LibraryProduct("liburcu-common", :urcu_common),
+    LibraryProduct("liburcu-bp", :urcu_bp)
```
Comment (giordano) [inline U/URCU/build_tarballs.jl:47]: It isn't a rule, but usually we use for the variables the same name as the corresponding file, including `lib`, like this:
```suggestion
    LibraryProduct("liburcu-memb", :liburcu_memb),
    LibraryProduct("liburcu-mb", :liburcu_mb),
    LibraryProduct("liburcu-qsbr", :liburcu_qsbr),
    LibraryProduct("liburcu-signal", :liburcu_signal),
    LibraryProduct("liburcu-cds", :liburcu_cds),
    LibraryProduct("liburcu", :liburcu),
    LibraryProduct("liburcu-common", :liburcu_common),
    LibraryProduct("liburcu-bp", :liburcu_bp)
```
Would you like to follow this tradition?

**Comment by hros** [comment]
implemented all suggested changes

File: U/URCU/build_tarballs.jl [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+    LibraryProduct("liburcu-common", :liburcu_common),
+    LibraryProduct("liburcu-bp", :liburcu_bp)
+]
+
+# Dependencies that must be installed before this package can be built
+dependencies = Dependency[
+]
+
+# Build the tarballs, and possibly a `build.jl` as well.
+build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies; julia_compat="1.6")
```
Comment (giordano) [inline U/URCU/build_tarballs.jl:43]: It looks like we need at least GCC 5 for aarch64:
```suggestion
build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies; julia_compat="1.6", preferred_gcc_version=v"5")
```

File: U/URCU/build_tarballs.jl [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+cd $WORKSPACE/srcdir/userspace-rcu/
+./bootstrap 
+./configure --prefix=${prefix} --build=${MACHTYPE} --host=${target}
+make -j${nproc}
+make install
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = supported_platforms(; experimental=true)
```
Comment (giordano) [inline U/URCU/build_tarballs.jl:24]: Ok, it looks like Windows isn't supported at all, so I guess we can skip it
```suggestion
platforms = filter!(!Sys.iswindows, supported_platforms(; experimental=true))
```

**Pull request merged**